bHY6yh-d61508/cod-bqe7y7_bHY6yh-d61508.xml

```diff
@@ -0,0 +1,140 @@
+      <editorialDecl>
+        <p>Encoding of this text has followed the recommendations of the LombardPress 1.0.0
+          guidelines for a diplomatic edition.</p>
+      </editorialDecl>
+    </encodingDesc>
+    <revisionDesc status="draft">
+      <listChange>
+        <change when="2023-12-19" status="draft" n="0.0.0">
+          <p>Created file for the first time.</p>
+        </change>
+      </listChange>
+    </revisionDesc>
+  </teiHeader>
+  <text xml:lang="la">
+    <front>
+      <div xml:id="starts-on">
+        <pb ed="#P" n="127-v"/>
+        <cb ed="#P" n="a"/>
+      </div>
+    </front>
+    <body>
+      <div xml:id="bHY6yh-d61508"><!-- p3l5-r2 -->
+        <head xml:id="bHY6yh-d61508-Hd1e108">Rubrica 2</head>
+        <head xml:id="bHY6yh-d61508-Hd1e111" type="question-title">De temperantia locutionis</head>
+        <p xml:id="bHY6yh-d61508-d1e114">
+          <lb ed="#P"/>De temperantia locutionis et
+          <lb ed="#P"/>oportuno sermone capitulo ii
+        </p>
+        <p xml:id="bHY6yh-d61508-d1e121">
+          <lb ed="#P"/><ref xml:id="bHY6yh-d61508-Rd1e118">Salomon in prouerbiis capitulo xviii</ref> <quote xml:id="bHY6yh-d61508-Qd1e127">Qui moderatur 
+          <cb ed="#P" n="b"/>
+          <lb ed="#P"/>sermones suos doctus et prudens est.</quote>
+        </p>
+        <p xml:id="bHY6yh-d61508-d1e133">
+          <lb ed="#P"/><ref xml:id="bHY6yh-d61508-Rd1e133">Seneca de moribus</ref> <quote xml:id="bHY6yh-d61508-Qd1e137">Magna res est vocis 
+          <lb ed="#P"/>et silentii temperamentum.</quote> 
+        </p>
+        <p xml:id="bHY6yh-d61508-d1e144">
+          <lb ed="#P"/><ref xml:id="bHY6yh-d61508-Rd1e146">Idem in prouerbiis</ref> <quote xml:id="bHY6yh-d61508-Qd1e148">Tene vocis et silentii temper a<lb ed="#P" break="no"/>mentum.</quote>
+        </p>
+        <p xml:id="bHY6yh-d61508-d1e158">
+          <lb ed="#P"/><ref xml:id="bHY6yh-d61508-Rd1e162">Auctor libri qui dicitur pamphilus capitulo iii</ref>
+          <lb ed="#P"/><quote xml:id="bHY6yh-d61508-Qd1e162">Nec nimium taceas nec verba superflua dicas</quote>
+        </p>
+        <p xml:id="bHY6yh-d61508-d1e168">
+          <lb ed="#P"/><ref xml:id="bHY6yh-d61508-Rd1e174">Bernardus de improbatione viciorum capitulo ii</ref> 
+          <lb ed="#P"/><quote xml:id="bHY6yh-d61508-Qd1e172">Verba nocent aptis si non moderentur habenis.
+          <lb ed="#P"/>Os quoque nec mutum / nec decet esse loquax.</quote>
+        </p>
+        <p xml:id="bHY6yh-d61508-d1e180">
+          <lb ed="#P"/><ref xml:id="bHY6yh-d61508-Rd1e189">henricus samariensis libro ii</ref> 
+          <lb ed="#P"/><quote xml:id="bHY6yh-d61508-Qd1e184">Stillet in ore fauus sed mente resultet 
+          <lb ed="#P"/>oliua / Et non sit totus sensus in ore tuus.</quote>
+        <p xml:id="bHY6yh-d61691-d1e114"><!--This is a chapter title-->
```

Comment:
This looks like the beginning of rubric 3. Perhaps this is a scribal error, who prematurely thought rubric 3 was starting. 

In any case, there is a missing closing `</p>` tag here.